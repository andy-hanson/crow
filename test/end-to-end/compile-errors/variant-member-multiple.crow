main void()
	_ nat v = r
	_ string v = r
	()

v[t] variant
r record nat v variant-member, string v variant-member
