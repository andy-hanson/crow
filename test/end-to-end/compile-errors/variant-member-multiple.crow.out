exit code: 1
stderr:
test/end-to-end/compile-errors/variant-member-multiple.crow 3:18-3:19 There are functions named 'r', but they do not match the expected return type and actual argument types.
Expected return type 'string v'.
Actual argument types: 
Candidates (with 0 arguments):
	'r nat64 v()' (from test/end-to-end/compile-errors/variant-member-multiple.crow line 7)
test/end-to-end/compile-errors/variant-member-multiple.crow 7:32-7:55 Type 'r' can't declare itself a member of the same variant 'v' multiple times.
Uncaught exception: Reached compile error
	at main
